crypto/fipsmodule/PQREADME.md

```diff
@@ -0,0 +1,65 @@
+# Post-Quantum Cryptography in AWS-LC
+
+AWS Cryptography focuses research and engineering efforts on the continuation of providing cryptographic security for our customers, while developing new cryptographic systems that exceed current customers’ demands and protect against projected future adversaries. This document contains notes about the design of the Post-Quantum (PQ) Cryptography provided by AWS-LC, and documentation on our current PQ integrations.
+
+In 2023 the U.S. Government passed the [Quantum Computing Cybersecurity Preparedness Act](https://www.congress.gov/bill/117th-congress/house-bill/7535/text), which creates requirements for government agencies to have a cryptographic inventory and plans to migrate to post-quantum (PQ) cryptography. These requirements extend to information technology providers like AWS. The NSA has also announced the [Commercial National Security Algorithm Suite (CNSA) 2.0](https://media.defense.gov/2022/Sep/07/2003071834/-1/-1/0/CSA_CNSA_2.0_ALGORITHMS_.PDF) that provides timelines for Cloud providers wishing to support National Security Systems, with support of PQ by 2025, and exclusively use PQ by 2033.
+
+## Relevant Standards
+To support these initiatives, the U.S. Department of Commerce’s National Institute of Standards and Technology (NIST) published three PQ algorithms as part of the Federal Information Processing Standards (FIPS):
+
+- [FIPS 203: Module-Lattice-Based Key-Encapsulation Mechanism Standard](https://csrc.nist.gov/pubs/fips/203/final)
+- [FIPS 204: Module-Lattice-Based Digital Signature Standard](https://csrc.nist.gov/pubs/fips/204/final)
+- [FIPS 205: Stateless Hash-Based Digital Signature Standard](https://csrc.nist.gov/pubs/fips/205/final)
+
+## AWS-LC Post-Quantum Algorithms
+
+AWS-LC provides the following post-quantum algorithms:
+
+### FIPS 203: Module-Lattice-Based Key-Encapsulation Mechanism (ML-KEM)
+
+| Algorithm        | Public Key (B) | Private Key (B) |Ciphertext(B) |
+|--------------------|-------------------|---------------------|-------------------|
+| ML-KEM-512   |                   800|                   1632|                   768|
```

Comment:
nit: Alignment is off, I find it easier to just retain a single space instead of trying to keep things aligned. Github will render it all the same anyway.

Reply:
Thanks, didn't know markdown would do this automatically for me! fixed in [6ee84ff](https://github.com/aws/aws-lc/pull/1932/commits/6ee84ffe30fa5c3a959b12e7f6feba51925507c0)